homeassistant/components/tplink/light.py

```diff
@@ -252,19 +238,19 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self.device.turn_off(transition=transition)
 
     @property
-    def min_mireds(self) -> int:
+    def min_color_temp_kelvin(self) -> int:
         """Return minimum supported color temperature."""
-        return kelvin_to_mired(self.device.valid_temperature_range.max)
+        return cast(int, self.device.valid_temperature_range.min)
```

Comment:
Unsure why casting here is required (mypy thinks it's `Any`) as the type is defined here: https://github.com/python-kasa/python-kasa/blob/master/kasa/smartbulb.py#L13

Potentially due to using `type: ignore` for the property `valid_temperature_range`?